paasta_tools/instance/kubernetes.py

```diff
@@ -528,6 +529,45 @@ def bounce_status(
     return status
 
 
+async def get_pods_for_service_instance_multiple_namespaces(
+    service: str,
+    instance: str,
+    kube_client: kubernetes_tools.KubeClient,
+    namespaces: Iterable[str],
+) -> Sequence[V1Pod]:
+    ret: List[V1Pod] = []
+
+    for coro in asyncio.as_completed(
+        [
+            kubernetes_tools.pods_for_service_instance(
+                service=service,
+                instance=instance,
+                kube_client=kube_client,
+                namespace=namespace,
+            )
+            for namespace in namespaces
+        ]
+    ):
+        ret.extend(await coro)
+
+    return ret
+
+
+def find_all_relevant_namespaces(
+    service,
+    instance,
+    kube_client,
+    job_config,
```

Comment:
nit: 
```suggestion
    service: str,
    instance: str,
    kube_client: kubernetes_tools.KubeClient,
    job_config: KubernetesDeploymentConfig,
```

(i'm guessing the type of `job_config` - might need to fix that one up)